custom_components/variable/__init__.py

```
@@ -49,16 +50,15 @@
 except ImportError:
-    from homeassistant.helpers.device import (
-        async_remove_stale_devices_links_keep_current_device,
-    )
+    from homeassistant.helpers.device import async_remove_stale_devices_links_keep_current_device
 
     def async_remove_helper_devices(
         hass: HomeAssistant,
         *,
         helper_config_entry_id: str,
         source_device_id: str | None,
         remove_all_devices: bool = False,
```

Ruff (0.16.1)

[warning] Unused function argument: `remove_all_devices`

(ARG001)